modules/upgrade/pages/db-migration.adoc

```diff
@@ -12,6 +12,21 @@ You cannot migrate directly from PostgreSQL{nbsp}9.4 to version 10.
 
 
 
+////
+== Background Information
+
+
+* {productname} 3.1.4, SLES 12 SP3, Postgres 9.6
+* {productname} 3.1.11, SLES 12 SP3, Postgres 9.6
+* {productname} 3.2.10, SLES 12 SP3, Postgres 9.6
+* {productname} 3.2.10, SLES 12 SP3, Postgres 10
+* {productname} 3.2.10, SLES 12 SP4, Postgres 10
+* {productname} 4.0.1, SLES 15 SP1, Postgres 10
```

Comment:
Better change thos to 4.0.0, SLES 15 SP1, Postgres 10 :-)

Reply:
ok, but it is currently commented...